dynacred/src/config.ts

```diff
@@ -0,0 +1,61 @@
+import * as toml from 'toml';
+
+/**
+ * Readonly container for the configuration
+ *
+ * TODO preciser name
+*/
+export class Config {
+    constructor(
+        readonly addresses: Addresses,
+    ) {}
+
+    /**
+    * Type containing both the addresses of the endpoints for servers and for
+    * clients
+    */
+    private class Addresses {
+        constructor(
+            readonly servers: string[],
+            readonly clients: string[],
+        ) {}
+    }
+}
+
+/**
+ * Load the Config from the server
+ */
+export default async function LoadConfig(): Promise<Config> {
+    const res = await fetch(window.location.origin + '/assets/conodes.toml');
+    if (!res.ok)
+        return Promise.reject(new Error(`on fetch config: ${res.status}: ${res.body}`));
```

Comment:
Does this pass the `tslint.json`?
1. Line 29 and others: single quotes instead of double quotes
2. `if` without the `{}`

Reply:
Did not, do now.

Reply:
+1 - I still remember my first braceless `if` that cost me 1 day of debugging. So everytime I see it again, I frown...